### ISSUE TITLE
Deprecation warnings in Gradle 8.8

### ISSUE THREAD

**Issue by rgrizzell** [boginw/migrations-gradle-plugin#69]
After recently upgrading to Gradle 8.8, the following warnings are emitted when using this plugin.

```
The org.gradle.api.plugins.Convention type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/8.8/userguide/upgrading_version_8.html#deprecated_access_to_conventions
The org.gradle.api.plugins.JavaPluginConvention type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/8.8/userguide/upgrading_version_8.html#java_convention_deprecation
```

**Comment by boginw** [comment]
Thank you for reporting this issue :)

I've tried to address the deprecation warnings in the latest release. It should now be resolved in version 0.4.0. 
Please upgrade to this version and let us know if you encounter any further issues.

Cheers

**Comment by rgrizzell** [comment]
Warnings have been resolved, thanks!
